### PR TITLE
Add HTML/JS for supporting switch sensor

### PR DESCRIPTION
**Pull request by wanghongjuan**
- Support to get the value of switch sensor when turn on/off the sensor

Signed-off-by: Hongjuan Wang hongjuan.wang@intel.com

### CODE REVIEW

**Comment by wanghongjuan** [comment]
@siovene PTAL

File: app/templates/sensor-switch.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<span class="sensor-text-value">
+	<div ng-show="sensor.data.properties.value !== undefined">
```
Comment (siovene) [inline app/templates/sensor-switch.html]: Indenting is off here.

File: app/templates/sensor-switch.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+        <span ng-show="sensor.data.properties.value === true" class="balanced">Turn On</span>
+        <span ng-show="sensor.data.properties.value === false" class="assertive">Turn Off</span>
```
Comment (siovene) [inline app/templates/sensor-switch.html]: As this is the current status of the button, do not use an action verb ("Turn") to express it. Either use "Turned on/off" or simply "On/Off".

**Comment by wanghongjuan** [comment]
Updated

**Pull request merged**